packages/components/src/Notification/ToastNotification/types.ts

```diff
@@ -0,0 +1,33 @@
+import { NotificationType } from "../types"
+
+type Modify<T, R> = Omit<T, keyof R> & R
+
+export type ToastNotification = {
+  id: string
+  type: NotificationType
+  title: string
+  message: React.ReactNode
+  onHide?: () => void
+  /**
+   * Removes the dismiss trigger. If this is true you will need to manage the removal of notifications manually.
```

Comment:
I don't think this is what was envisioned for this prop (for people implementing to find their own way to hide it). I reckon it was probably just implemented because the other notifications had it as a way of saying this notification will never disappear. Which probably should have never been an option for toasts. I reckon we should just get rid of this

Reply:
That would explain why the JSX version of the component has a `hideCloseIcon` prop as well 🤔

Reply:
So do think both `hideCloseIcon` and `persistent` should be removed entirely?

Reply:
Let's clarify the purpose of Toast Notifications with the designers to make sure it aligns with their understanding of the feature set in relation to the other "toasts" we have. But doing anything about it should be out of scope for the migration.

Reply:
Sounds good, will leave as is for now